package.json

```diff
@@ -2,7 +2,7 @@
   "name": "adze",
   "description": "A universal, modern, and extendable logging library for JavaScript and TypeScript.",
   "version": "2.0.0",
-  "main": "index.js",
+  "main": "dist/index.js",
```

Comment:
This change will break how semantic release currently builds and releases the package. This is an issue I wanted to address anyway, because putting index.js into the release makes it easier to test locally as the pathing will be the same.

To fix this, we'll just need to make a small change to the github actions to make sure dist is in the correct place when it is built and released.